test/spoke/integration/BalanceSheet.t.sol

```diff
@@ -87,316 +31,4 @@ contract BalanceSheetTest is BaseTest {
         assertEq(balanceSheet.wards(address(messageDispatcher)), 1);
```

Comment:
What we're testing here is the deployment scripts, not things related to the contract. I think we should move all these tests to an common test inside `test/script`. WDYT?